src/components/userInfo/userInfo.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { describe, test, expect } from 'vitest'
+import { screen } from '@testing-library/react'
+import { renderAuthenticated } from '../../setupTests'
+import UserInfo from './userInfo'
+
+describe('<UserInfo />', () => {
+  test('UserInfo displays user name and email', () => {
```

Comment:
Again, I won't add a test for if auth is loading because there wouldn't be a user whose info we should make sure isn't there.